docs/userguide/dockerimages.md

```diff
@@ -1,28 +1,26 @@
 <!--[metadata]>
 +++
-title = "Get started with images"
+title = "Build your own images"
 description = "How to work with Docker images."
 keywords = ["documentation, docs, the docker guide, docker guide, docker, docker platform, virtualization framework, docker.io, Docker images, Docker image, image management, Docker repos, Docker repositories, docker, docker tag, docker tags, Docker Hub,  collaboration"]
 [menu.main]
-parent = "smn_images"
-weight = 1
+parent = "smn_containers"
+weight = -4
 +++
 <![end-metadata]-->
 
-# Get started with images
+# Build your own images
 
-In the [introduction](../introduction/understanding-docker.md) we've discovered that Docker
-images are the basis of containers. In the
-[previous](dockerizing.md) [sections](usingdocker.md)
-we've used Docker images that already exist, for example the `ubuntu`
-image and the `training/webapp` image.
+Docker images are the basis of containers. Each time you've used `docker run`
+you told it which image you wanted. In the previous sections of the guide you
+used Docker images that already exist, for example the `ubuntu` image and the
+`training/webapp` image.
 
-We've also discovered that Docker stores downloaded images on the Docker
-host. If an image isn't already present on the host then it'll be
-downloaded from a registry: by default the
-[Docker Hub Registry](https://registry.hub.docker.com).
+You also discovered that Docker stores downloaded images on the Docker host. If
+an image isn't already present on the host then it'll be downloaded from a
+registry: by default the [Docker Hub Registry](https://registry.hub.docker.com).
 
-In this section we're going to explore Docker images a bit more
+In this section you're going to explore Docker images a bit more
```

Comment:
I was more interested in getting rid of the royal we.  I have to go back and rewrite/restructure everything in this area but time is short for the 1.0 release. So, I'm not going to change every bit of language --- much as I would have liked to.